_overviews/core/custom-collections.md

```diff
@@ -576,38 +837,84 @@ this is why we always get a `Vector` as a result.
-      override protected def newSpecificBuilder: mutable.Builder[Base, RNA2] = // as before
-      
-      // Overloading of `appended`, `prepended`, `appendedAll`,
-      // `prependedAll`, `map`, `flatMap` and `concat` to return an `RNA2`
-      // when possible
-      def concat(suffix: IterableOnce[Base]): RNA2 =
-        fromSpecific(iterator ++ suffix.iterator)
-      // symbolic alias for `concat`
-      @inline final def ++ (suffix: IterableOnce[Base]): RNA2 = concat(suffix)
-      def appended(base: Base): RNA2 =
-        fromSpecific(new View.Appended(this, base))
-      def appendedAll(suffix: IterableOnce[Base]): RNA2 =
-        concat(suffix)
-      def prepended(base: Base): RNA2 = 
-        fromSpecific(new View.Prepended(base, this))
-      def prependedAll(prefix: IterableOnce[Base]): RNA2 =
-        fromSpecific(prefix.iterator ++ iterator)
-      def map(f: Base => Base): RNA2 =
-        fromSpecific(new View.Map(this, f))
-      def flatMap(f: Base => IterableOnce[Base]): RNA2 =
-        fromSpecific(new View.FlatMap(this, f))
-    }
+{% tabs RNA2_1 class=tabs-scala-version %}
+{% tab 'Scala 2' for=RNA2_1 %}
+~~~ scala
+import scala.collection.{ View, mutable }
+import scala.collection.immutable.{ IndexedSeq, IndexedSeqOps }
+
+final class RNA2 private (val groups: Array[Int], val length: Int)
+  extends IndexedSeq[Base] with IndexedSeqOps[Base, IndexedSeq, RNA2] {
+
+  import RNA2._
+
+  def apply(idx: Int): Base = // as before
+  override protected def fromSpecific(coll: IterableOnce[Base]): RNA2 = // as before
+  override protected def newSpecificBuilder: mutable.Builder[Base, RNA2] = // as before
+  override def empty: RNA2 = // as before
+  override def className = "RNA2"
+  
+  // Overloading of `appended`, `prepended`, `appendedAll`,
+  // `prependedAll`, `map`, `flatMap` and `concat` to return an `RNA2`
+  // when possible
+  def concat(suffix: IterableOnce[Base]): RNA2 =
+    fromSpecific(iterator ++ suffix.iterator)
+  // symbolic alias for `concat`
+  @inline final def ++ (suffix: IterableOnce[Base]): RNA2 = concat(suffix)
+  def appended(base: Base): RNA2 =
+    fromSpecific(new View.Appended(this, base))
+  def appendedAll(suffix: IterableOnce[Base]): RNA2 =
+    concat(suffix)
+  def prepended(base: Base): RNA2 = 
+    fromSpecific(new View.Prepended(base, this))
+  def prependedAll(prefix: IterableOnce[Base]): RNA2 =
+    fromSpecific(prefix.iterator ++ iterator)
+  def map(f: Base => Base): RNA2 =
+    fromSpecific(new View.Map(this, f))
+  def flatMap(f: Base => IterableOnce[Base]): RNA2 =
+    fromSpecific(new View.FlatMap(this, f))
+}
+~~~
+{% endtab %}
+{% tab 'Scala 3' for=RNA2_1 %}
+~~~ scala
+import scala.collection.{ View, mutable }
+import scala.collection.immutable.{ IndexedSeq, IndexedSeqOps }
+
+final class RNA2 private (val groups: Array[Int], val length: Int)
+  extends IndexedSeq[Base], IndexedSeqOps[Base, IndexedSeq, RNA2]:
+
+  import RNA2._
```

Comment:
```suggestion
  import RNA2.*
```

Reply:
It look like I forget a lot of stars... Thanks